avaframe/runCom6Scarp.py

```diff
@@ -0,0 +1,10 @@
+#RunScarp.py
```

Comment:
Please look at one of our runscripts, for example AvaFrame/avaframe/runIn1RelInfo.py and copy that structure (with the main etc). This is necessary to be easily callable from the QGis connector. 

I also suggest using a different way of getting the config file. i.e. instead of giving it directly, import the module scarp and then load the config. This way, the normal logic of possible usage of local_cfg.ini etc is observered. If you are not sure about what is needed to do this, let me know and I'm happy to help.